src/self_driving/pilot_util.cpp

```diff
@@ -115,13 +241,31 @@ void PilotUtil::InferenceWithFeatures(
     }
     inference_result.emplace(ou_map_it.first, res.first);
   }
+
+  // populate pipeline_to_prediction using pipeline_to_ou_position and inference_result
+  std::vector<std::vector<double>> ppl_res;
```

Comment:
nit: `ppl_res`->`pipeline_result`. Change other `ppl` in this function to `pipeline` as well. In general, avoid abbreviation if it's not super long.